src/Compilers/CSharp/Portable/FlowAnalysis/UnassignedAddressTakenVariablesWalker.cs

```diff
@@ -12,7 +12,7 @@ namespace Microsoft.CodeAnalysis.CSharp
     /// An analysis that computes the set of variables that may be used
     /// before being assigned anywhere within a method.
     /// </summary>
```

Comment:
This comment is simply incorrect.

Reply:
Moved to #31483

---
In reply to: [236839367](https://github.com/dotnet/roslyn/pull/31389#discussion_r236839367) [](ancestors = 236839367)